src/popup/locales/de.json

```diff
@@ -74,7 +74,7 @@
       "SpendTx": "Spend",
       "ContractCreateTx": "Contract create",
       "ContractCallTx": "Contract call",
-      "NamePreclaimTx": "Preclaim name",
+      "namePreclaimTx": "Preclaim name",
```

Comment:
I think we should not keep the untranslated string in translation files. Maybe just remove the whole untranslated block to avoid having to update all files every time something changes? Also the surrounding keys does not match the EN keys.

Reply:
Good catch. This diff was by mistake. Rolling back. Thanks